test/integration/test.speech_to_text.js

```diff
@@ -380,5 +380,34 @@ describe('speech_to_text_integration', function() {
       speech_to_text.deleteCustomization({ customization_id: customization_id }, done);
       customization_id = null;
     });
+
+    it('registerCallback()', function(done) {
+      const params = { callback_url: 'http://www.callback.fr/results', user_secret: 'ThisIsMySecret' };
+      speech_to_text.registerCallback(params, done);
+    });
+
+    it('createRecognitionJob()', function(done) {
+      const params = {
+        audio: fs.createReadStream(__dirname + '/../resources/weather.wav'),
+        content_type: 'audio/l16;rate=41100',
+        callback_url: 'http://www.callback.fr/results',
+        user_token: 'ThisIsMySecret',
+        event: 'recognitions.completed',
+        results_ttl: '60'
+      };
+      speech_to_text.createRecognitionJob(params, done);
+    });
+
+    it('getRecognitionJobs()', function(done) {
+      speech_to_text.getRecognitionJobs(done);
+    });
+
+    it('getRecognitionJob()', function(done) {
+      speech_to_text.getRecognitionJob({ id: 'Job01' }, done);
+    });
+
+    it('deleteRecognitionJob()', function(done) {
+      speech_to_text.getRecognitionJob({ id: 'Job01' }, done);
```

Comment:
Also, the job id should correspond to the one created earlier. (And, if necessary, the test should wait for it to finish before deleting it.)